src/content/developers/docs/oracles/index.md

```diff
@@ -174,6 +174,64 @@ contract RandomNumberConsumer is VRFConsumerBase {
 }
 ```
 
+### Chainlink Keepers {#chainlink-keepers}
+
+Smart contracts alone can't trigger or initiate their own functions at arbitrary times or under arbitrary conditions. State change will only occur when a transaction is initiated by another account (such as user, oracle, or contract). The [Chainlink Keeper Network](https://docs.chain.link/docs/chainlink-keepers/introduction/) provides options for smart contracts to outsource regular maintenance tasks in a trust minimized and decentralized manner.
+
+In order to use Chainlink Keepers, smart contract needs to implement [KeeperCompatibleInterface](https://docs.chain.link/docs/chainlink-keepers/compatible-contracts/) which consits of two functions:
+
+- `checkUpkeep` - Checks if the contract requires work to be done.
+- `performUpkeep` - Performs the work on the contract, if instructed by checkUpkeep.
+
+The example below represents a simple counter contract. Each time performUpkeep is called, it increments its counter by one. [One can deploy the following code using Remix](https://remix.ethereum.org/#url=https://docs.chain.link/samples/Keepers/KeepersCounter.sol)
+
+```javascript
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+// KeeperCompatible.sol imports the functions from both ./KeeperBase.sol and
+// ./interfaces/KeeperCompatibleInterface.sol
+import "@chainlink/contracts/src/v0.7/KeeperCompatible.sol";
+
+contract Counter is KeeperCompatibleInterface {
+    /**
+    * Public counter variable
+    */
+    uint public counter;
+
+    /**
+    * Use an interval in seconds and a timestamp to slow execution of Upkeep
+    */
+    uint public immutable interval;
+    uint public lastTimeStamp;
+
+    constructor(uint updateInterval) {
+      interval = updateInterval;
+      lastTimeStamp = block.timestamp;
+
+      counter = 0;
+    }
+
+    function checkUpkeep(bytes calldata /* checkData */) external override returns (bool upkeepNeeded, bytes memory /* performData */) {
+        upkeepNeeded = (block.timestamp - lastTimeStamp) > interval;
+        // We don't use the checkData in this example. The checkData is defined when the Upkeep was registered.
+    }
+
+    function performUpkeep(bytes calldata /* performData */) external override {
+        lastTimeStamp = block.timestamp;
+        counter = counter + 1;
+        // We don't use the performData in this example. The performData is generated by the Keeper's call to your checkUpkeep function
+    }
+}
+```
+
+After one deploys a contract, it does not automatically begin to receive Upkeeps. The final step is [Upkeep Registration](https://docs.chain.link/docs/chainlink-keepers/register-upkeep/).
```

Comment:
```suggestion
After deploying a contract, it does not automatically begin to receive Upkeeps. The final step is [Upkeep Registration](https://docs.chain.link/docs/chainlink-keepers/register-upkeep/).
```

Reply:
This feels a little incomplete to me. At the very least we should explicitly state what `receiving an Upkeep` means here instead of expecting the reader to deduce it from the surrounding context.

Reply:
Hey @minimalsm, thanks for the feedback! You are right, this is a little misleading. How does the next sentence sound to you?

> After deploying a Keeper-compatible contract, you must register it for [Upkeep](https://docs.chain.link/docs/chainlink-keepers/register-upkeep/).

Reply:
@andrejrakic yeah this sounds better but I'd explain why.

> After deploying a Keeper-compatible contract, you must register the contract for Upkeep to....

Reply:
@minimalsm good point! Do you think the following sentence is clear to the average reader?
> After deploying a Keeper-compatible contract, you must register the contract for [Upkeep](https://docs.chain.link/docs/chainlink-keepers/register-upkeep/) and fund it with LINK, to notify the Keeper Network about your contract, so your work is performed continuously.